src/main.rs

```diff
@@ -604,6 +605,15 @@ impl Context {
         Ok(())
     }
 
+    fn invalidate_manifests_txt(&self) -> Result<(), Error> {
```

Comment:
Can you instead do a `fn invalidate(paths: &[&str])` to replace both `invalidate_releases` and `invalidate_manifests_txt`?